tools/points2labelimage/points2label.xml

```diff
@@ -4,7 +4,7 @@
         <import>creators.xml</import>
         <import>tests.xml</import>
         <token name="@TOOL_VERSION@">0.4.1</token>
-        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@VERSION_SUFFIX@">1</token>
     </macros>
     <creator>
```

Comment:
Feel free to add yourself to the `<creator>` section :)

Reply:
thanks